server/tinylicious/package.json

```diff
@@ -86,7 +86,7 @@
 		"@types/mocha": "^9.1.1",
 		"@types/morgan": "^1.7.35",
 		"@types/nconf": "^0.10.0",
-		"@types/node": "^16.18.16",
+		"@types/node": "^18.17.1",
```

Comment:
We're not responsible for T9s, so I'd revert these changes till the team who owns it can tackle this and do their validation.

Reply:
Otherwise, if we do want to update T9s, then we need to do more updates, like the `engines` field in the t9s/package.json, and the t9s build pipelines. Also the T9s/devcontainer/dockerfile is still on Node 14, so we would probably want to update that as well.

Reply:
It looks like build is failing for T9s due to these changes, so it's probably best to revert for now.

Reply:
Changed back.